packages/venia-concept/jest.config.js

```diff
@@ -1,8 +1,19 @@
+const { join } = require('path');
+
 module.exports = {
     displayName: 'Venia Concept',
     browser: true,
     moduleNameMapper: {
         '\\.css$': 'identity-obj-proxy',
-        '^src/(.+)': '<rootDir>/src/$1'
-    }
+        // Mirrors webpack alias to resolve from 'src'
+        '^src/(.+)': '<rootDir>/src/$1',
+        // Re-write imports to Peregrine to ensure they're not pulled from the
+        // (possibly outdated) build artifacts on disk in `dist`.
+        // Ideally this rule would be in the root Jest config, but Jest's config
```

Comment:
One tiny thing: Please put something like `TODO: review when Jest revs` here so that these tech debt items from dependencies are searchable.

Reply:
Done